client/components/profilePage/friendRecommendations.jsx

```diff
@@ -12,21 +12,21 @@ export default function FriendRecommendations({user}) {
         <p>What friends are recommending</p>
         </Col>
       </Row>
-      <Row>
-        <Col>
-          <Card className="recommendation-cards">
-            <p>Yo! This Dr. Seuss Book is quite the masterpiece! Never knew I was so gifted in selecting books!</p>
-            <p>Jerrick Ravelo</p>
-          </Card>
-        </Col>
-      </Row>
-      <Row>
-        <Col>
-          <Card className="recommendation-cards">
-            <p>Jason Fleming recommends...</p>
-          </Card>
-        </Col>
-      </Row>
+      {reviewList.length > 0 &&
+      reviewList.slice(0,3).map((friend) =>{
```

Comment:
Look into dynamically choosing the amount of reviews to render after meeting today